channeldb/channel.go

```diff
@@ -729,7 +729,7 @@ type OpenChannel struct {
 	RevocationKeyLocator keychain.KeyLocator
 
 	// TODO(roasbeef): eww
-	Db *DB
+	Db *ChannelStateDB
```

Comment:
I think it's finally time to remove that comment above :sweat_smile:

Reply:
I still think it's somewhat eww :smiley: